traitsui/tests/test_color_column.py

```diff
@@ -38,11 +42,9 @@ class TestColorColumn(TestCase):
     def test_color_column(self):
         # Behaviour: column ui should display without error
 
-        with store_exceptions_on_all_threads():
-            d1 = MyEntry(name="a", value=2, color=(1.0, 0.3, 0.1))
-            d2 = MyEntry(name="b", value=3, color=(0.1, 0.0, 0.9))
-            data = MyData(data_list=[d1, d2])
-
-            ui = data.edit_traits()
-
-            ui.dispose()
```

Comment:
This does not actually need changing, but it also does not hurt. Happy to revert if requested.

Reply:
There are more instances of this pattern in `ui_editors/test_data_frame.py`, `editors/test_tree_editor.py` (which are already being modified in this PR) and in `qt4/tests_test_ui_panel.py`. So if changing the pattern, I think it's best to change it everywhere.

Reply:
I will revert this. I don't think this needs changing.

Reply:
(On the other hand, I would also be in favour of making patterns more consistent, but maybe best done in a separate PR.)